<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminWindowClass</class>
 <widget class="QMainWindow" name="AdminWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>889</width>
    <height>634</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>reretry</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QListWidget" name="allEventsListWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="titleLabel">
            <property name="text">
             <string>Title</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="titleInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="linkLabel">
            <property name="text">
             <string>Link</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="linkInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="descriptionLabel">
            <property name="text">
             <string>Description</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextEdit" name="descriptionInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="attendanceLabel">
            <property name="text">
             <string>Attendance</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="attendanceInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="dateHourLabel">
            <property name="text">
             <string>Date and time</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateTimeEdit" name="dateHourInput"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="addButton">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="deleteButton">
        <property name="text">
         <string>Delete what you selected</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="updateButton">
        <property name="text">
         <string>What you selected becomes what you wrote</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="openChart">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>300</height>
       </size>
      </property>
      <property name="font">
       <font>
        <bold>true</bold>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>OpenHandCursor</cursorShape>
      </property>
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Chart</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>889</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="reretry.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>deleteButton</sender>
   <signal>clicked()</signal>
   <receiver>AdminWindowClass</receiver>
   <slot>deleteSong()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>217</x>
     <y>552</y>
    </hint>
    <hint type="destinationlabel">
     <x>418</x>
     <y>564</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>deleteSong()</slot>
 </slots>
</ui>
